README.md

```diff
@@ -311,14 +307,11 @@ skate('my-element', {
       // a custom value, specify a getter.
       set: function (newValue, oldValue) {},
```

Comment:
This doesn't match the signature of the call to `opts.set(this, changeData)` in `src/lifecycle/property.js`.